src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/WrappedGr.cpp

```diff
@@ -0,0 +1,304 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/WrappedGr.hpp"
+
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/DataVector.hpp"        // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"     // IWYU pragma: keep
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Minkowski.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp"
+#include "PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+// IWYU pragma: no_forward_declare ::Tags::deriv
+
+/// \cond
+namespace GeneralizedHarmonic {
+namespace Solutions {
+// Preprocessor logic to avoid defining variables() functions for
+// tags other than the three the wrapper adds (i.e., other than
+// gr::Tags::SpacetimeMetric, gr::Tags::Lapse, and gr::Tags::Shift.
```

Comment:
Shouldn't `Lapse` and `Shift` be `GeneralizedHarmonic::Tags::Phi` and `GeneralizedHarmonic::Tags::Pi`? And you're missing a closing paren :)